tests/Benchmarks/Encodings/AsciiDecoding.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text;
+using System.Text.Utf8;
+using BenchmarkDotNet.Attributes;
+
+namespace Benchmarks.Encodings
+{
+    public class AsciiDecoding
+    {
+        private byte[] bytes;
+
+        [Params("/plaintext", "text/plain,text/html;q=0.9,application/xhtml+xml;q=0.9,application/xml;q=0.8,*/*;q=0.7")]
+        public string Text;
+
+        [GlobalSetup]
+        public void Setup() => bytes = Encoding.ASCII.GetBytes(Text);
+
+        [Benchmark]
+        public string AsciiToStringPrimitives() => System.Buffers.Text.Encodings.Ascii.ToUtf16String((ReadOnlySpan<byte>)bytes);
+
+        [Benchmark]
```

Comment:
Should this be the baseline, i.e. existing CLR implementation being the baseline.